### PR TITLE
P1

### PR DESCRIPTION
**Pull request by tylerharter**

### CODE REVIEW

File: fall18/p1/main.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+# put something in the parenthesis here to get an answer
+print()
+print()
+
+############################################################
+print('Problem 10')
+
+print('three plus four is:')
+print("3" + "4") # fix this to get 7
```
Comment (EffingerB) [inline fall18/p1/main.py]: Would it be appropriate for students to just do print(7) or do we want to make sure they specifically do print(4 + 3)

File: fall18/p1/README.md [hunk 2]
```diff
@@ -1,3 +1,57 @@
 # Project 1
 
-More details coming soon.
+In this project, you'll learn about types, operators, and boolean
+logic.  Two start, download these two files:
```
Comment (adalbertgerald) [inline fall18/p1/README.md]: Typo: Two start -> To start

File: fall18/p1/README.md [hunk 3]
```diff
@@ -1,3 +1,57 @@
...
+
+* [main.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p1/main.py)
+* [test.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p1/test.py)
+
+You will change main.py and hand it in.  You should not change
+test.py, and you should not hand it in; it's only purpose is tell you
```
Comment (adalbertgerald) [inline fall18/p1/README.md]: Typo: "it's only purpose is tell you" -> "it's only purpose is to tell you"

File: fall18/p1/README.md [hunk 4]
```diff
@@ -1,3 +1,57 @@
...
+This means if you turn in main.py now, you'll get 10% for your score.
+Pretty good for having done nothing yet, no?
+
+You would get 10% because there are 20 problems, each worth 5%, and we
+have done problems 1 and 3 for you.  You can see this because the
+output above says "PASS" by them.  Your goal is to get more points by
+getting test.py to print "PASS" by more problems.
+
+The rest of the instructions are in main.py, so open it up in your
+editor (probably PyCharm), and starting coding.  Have fun, and run
```
Comment (adalbertgerald) [inline fall18/p1/README.md]: Grammar: "starting coding" -> "start coding"

File: fall18/p1/main.py [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+print('the type of "5.0" is:') # note the quotes!
+
+# put something in the parenthesis here to get an answer
+print()
+print()
+
+############################################################
+print('Problem 7')
+
+print('the type "True" is:')
```
Comment (adalbertgerald) [inline fall18/p1/main.py]: Change this as 'the type of True is: ' ("of" is missing which leads to test failing).
Also, adding a comment saying "# note the quotes!" here too might be helpful.

File: fall18/p1/main.py [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+############################################################
+print('Problem 19')
+
+number = 98
+print('There are three winners, with numbers 145, 98, and 256!')
+print('Do we have a winning number?')
+
+# aww, why didn't win?  Fix the following so that it will show True
+# for 98, and will also show True if we were to replace 98 with 145 or
+# 456
```
Comment (adalbertgerald) [inline fall18/p1/main.py]: In comment: 456 -> 256

**Pull request merged**